File: SDKs/amazonaws.com/amplifybackend/2020-08-11/go/pkg/models/shared/gettokenresponse.go
package shared

type GetTokenResponse struct {
	AppID         *string `json:"AppId,omitempty"`
	ChallengeCode *string `json:"ChallengeCode,omitempty"`
	SessionID     *string `json:"SessionId,omitempty"`
	TTL           *string `json:"Ttl,omitempty"`
}

File: SDKs/amazonaws.com/amplifybackend/2020-08-11/typescript/src/sdk/models/shared/gettokenresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetTokenResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=AppId" })
  appId?: string;

  @SpeakeasyMetadata({ data: "json, name=ChallengeCode" })
  challengeCode?: string;

  @SpeakeasyMetadata({ data: "json, name=SessionId" })
  sessionId?: string;

  @SpeakeasyMetadata({ data: "json, name=Ttl" })
  ttl?: string;
}
